pkg/networking/middleware/response_test.go

```diff
@@ -75,9 +75,8 @@ func Test_ResponseMiddleware(t *testing.T) {
 			snykErr := snyk_errors.Error{}
 			url := fmt.Sprintf("%s/%d", server.URL, code)
 			req := buildRequest(url)
-			res, err := rt.RoundTrip(req)
+			_, err := rt.RoundTrip(req)
```

Comment:
Question: is it right to ignore the response here or should we express the intention that the response is not nil

Reply:
<img src="https://media2.giphy.com/media/v1.Y2lkPWJkM2VhNTdld21zZzN0MmozZ2xqbzRjZmFzd2NudzR5anhlMTNkZm5zc29iOG03OCZlcD12MV9naWZzX3NlYXJjaCZjdD1n/KSyrh7IhyW28HKjmTn/giphy.gif"/>